modules/services/espanso.nix

```diff
@@ -42,10 +59,31 @@ in
       package = mkOption {
         type = types.package;
         description = "Which espanso package to use";
-        default = pkgs.espanso;
+        default = if isDarwin || cfg.x11Support then pkgs.espanso else pkgs.espanso-wayland;
         defaultText = literalExpression "pkgs.espanso";
       };
 
+      package-wayland = mkOption {
+        type = types.package;
+        description = "Which espanso package to use when running under wayland on linux";
+        default = if isLinux && cfg.waylandSupport then pkgs.espanso-wayland else pkgs.espanso;
+        defaultText = literalExpression "pkgs.espanso-wayland";
+      };
```

Comment:
This makes no sense, to me. We already have a package option. You can change the desired package there.

Reply:
what would be your suggestion, for espanso to seamlessly work on both x11 and wayland by default.

Reply:
This is to customise the wayland version of the package, say a patched version of espanso-wayland.

Reply:
I think this should just be strictly a default of `espanso-wayland`, at least. Makes no sense for the wayland package to not be wayland compatible by default, ever.

Reply:
yes these `default` conditions for both `package` and `package-wayland` are actually redundant I realised. the package-bin does handle all of them fine.